test-net.json

```diff
@@ -1,3 +1,61 @@
-[
-
-]
+var tokens = {
+  "name": "Default List",
+  "tokens": [
+    {
+      "name": "NEAR",
+      "type": "Native token",
+      "address": "",
+      "symbol": "NEAR",
+      "decimals": 24,
+      "logoURI": "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/near/info/logo.png"
```

Comment:
we renamed this attribute to `logoURL`

Reply:
ok

Reply:
you will need to assign loaded JSON to a variable. You can use `fetch` for that:
```
fetch("https://path_to_list.json")
.then(response => {
   tokens = response.json();
})
```